<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bernsenThresholdDialog</class>
 <widget class="QDialog" name="bernsenThresholdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bernsen Threshold</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="minContrastLabel">
     <property name="text">
      <string>min contrast (I_max - I_min):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <layout class="QHBoxLayout" name="radiusLayout">
     <item>
      <widget class="QSpinBox" name="radiusSpinBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pxLabel">
       <property name="text">
        <string>px</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="bgTypeLayout">
     <item>
      <widget class="QComboBox" name="bgTypeComboBox">
       <item>
        <property name="text">
         <string>dark</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bright</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="bgTypeLabel">
     <property name="text">
      <string>background type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="radiusLabel">
     <property name="text">
      <string>radius:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout" name="minContrastLayout">
     <item>
      <widget class="QSpinBox" name="minContrastSpinBox">
       <property name="maximum">
        <number>254</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="printToConsoleCheckBox">
     <property name="text">
      <string>print mask to console</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="convertToGSCheckBox">
     <property name="text">
      <string>Convert RGB to grayscale</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>bernsenThresholdDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>bernsenThresholdDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
